python_modules/libraries/dagster-slack/setup.py

```diff
@@ -31,10 +31,8 @@ def get_version():
         packages=find_packages(exclude=["test"]),
         install_requires=[
             f"dagster{pin}",
-            "slackclient>=2,<3",
-            # resolve issue with aiohttp pin of chardet for aiohttp<=3.7.3, req'd by slackclient
-            # https://github.com/dagster-io/dagster/issues/3539
-            "chardet<4.0",
+            "slack_sdk",
+            "aiohttp",
```

Comment:
not obvious why this is added - does slack need it but not declare it?

Reply:
`slack_sdk` doesn't but it turns out I missed a bunch of scattered places throughout the repo that still used `slack` - I've fixed those up so we don't need to require this anymore.